app/controllers/logs_controller.rb

```diff
@@ -2,41 +2,46 @@ class LogsController < ApplicationController
   before_action :log_owner, only: [:index, :stockrooms]
 
   def index
-    @logs = Log.where(id: @user_logs).paginate(page: params[:page], per_page: 10)
+    # @logs = Log.where(id: @user_logs).paginate(page: params[:page], per_page: 10)
+    @logs = current_user.logs.paginate(page: params[:page], per_page: 10)
```

Comment:
`current_user.logs`
that is a database relationship. meaning the id of user is attached to the logs. (you will see how it is setup on the model USER)